src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -352,3 +352,24 @@ def test_aks_disable_addons_confcom_addon(self, resource_group, resource_group_l
             self.check('addonProfiles.accsgxdeviceplugin.enabled', False),
             self.check('addonProfiles.accsgxdeviceplugin.config', None)
         ])
+
+    @live_only()
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_stop_and_start(self, resource_group, resource_group_location):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name}'
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded'),
+        ])
+
+        stop_cmd = 'aks stop --resource-group={resource_group} --name={name}'
+        self.cmd(stop_cmd)
```

Comment:
should we add some validation here on the output?

Reply:
The only validation to be done is the status code, which is already done by `self.cmd`